### PR TITLE
add recording of MemAvailable

### PR DESCRIPTION
**Pull request by spacetc62**
https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=34e431b0ae398fc54ea69ff85ec700722c9da773

### CODE REVIEW

**Comment by janxious** [comment]
I'm thinking there could be a test for this, but :+1:

**Comment by mediocretes** [comment]
:+1: but also, mixed quotes?  I know you matched the file style, but this file, man.

File: lib/instrumental_tools/system_inspector/linux.rb [hunk 1]
```diff
@@ -96,6 +97,11 @@ def self.memory
 
       }
 
+      if available
+        stats_to_record['memory.available_mb']      = available / 1024
+        stats_to_record['memory.available_percent'] = (available / total) * 100
```
Comment (jqr) [inline lib/instrumental_tools/system_inspector/linux.rb]: `available.to_f` should give a more accurate result on machines with small amounts of ram.

Comment (janxious) [inline lib/instrumental_tools/system_inspector/linux.rb]: `available` has already been `to_f`'ed when it is set above, so oversight or am I missing something?

Comment (jqr) [inline lib/instrumental_tools/system_inspector/linux.rb]: Ah, I missed that section.

**Comment by janxious** [comment]
@mediocretes you should merge this and bump everything and build new packages now that you know how to do that dance.

**Comment by mediocretes** [comment]
Who's picture is that next to michael's name?!

**Comment by tonydewan** [comment]
Some lumberjack hacker, I'd guess. We should alert him that his github account was compromised

**Comment by spacetc62** [comment]
I am totally Michael and not an imposter, I swear

**Pull request merged**